data/sports/football/world_cup_history.json

```diff
@@ -0,0 +1,173 @@
+    "year": "1950",
+    "organizer": "Brazil",
+    "winner": "Uruguay",
+    "runners-up": "Brazil",
+    "third":"Sweden",
+    "fourth":"Spain"
+  },
+  {
+    "year": "1954",
+    "organizer": "Switzerland",
+    "winner": "Germany FR",
+    "runners-up": "Hungary",
+    "third":"Austria",
+    "fourth":"Uruguay"
+  },
+  {
+    "year": "1958",
+    "organizer": "Sweden",
+    "winner": "Brazil",
+    "runners-up": "Sweden",
+    "third":"France",
+    "fourth":"Germany FR"
+  },
+  {
+    "year": "1962",
+    "organizer": "Chile",
+    "winner": "Brazil",
+    "runners-up": "Checoslovakia",
+    "third":"Chile",
+    "fourth":"Yugoslavia"
+  },
+  {
+    "year": "1966",
+    "organizer": "England",
+    "winner": "England",
+    "runners-up": "Germany FR",
+    "third":"Portugal",
+    "fourth":"Soviet Union"
+  },
+  {
+    "year": "1970",
+    "organizer": "Mexico",
+    "winner": "Brazil",
+    "runners-up": "Italy",
+    "third":"Germany FR",
+    "fourth":"Uruguay"
+  },
+  {
+    "year": "1974",
+    "organizer": "Germany FR",
+    "winner": "Germany FR",
+    "runners-up": "Netherlands",
+    "third":"Poland",
+    "fourth":"Brazil"
+  },
+  {
+    "year": "1978",
+    "organizer": "Argentina",
+    "winner": "Argentina",
+    "runners-up": "Netherlands",
+    "third":"Brazil",
+    "fourth":"Italy"
+  },
+  {
+    "year": "1982",
+    "organizer": "Spain",
+    "winner": "Italy",
+    "runners-up": "Germany",
+    "third":"Poland",
+    "fourth":"France"
+  },
+  {
+    "year": "1986",
+    "organizer": "Mexico",
+    "winner": "Argentina",
+    "runners-up": "Germany FR",
+    "third":"France",
+    "fourth":"Belgium"
+  },
+  {
+    "year": "1990",
+    "organizer": "Italy",
+    "winner": "Germany FR",
+    "runners-up": "Argentina",
+    "third":"Italy",
+    "fourth":"England"
+  },
+  {
+    "year": "1994",
+    "organizer": "USA",
+    "winner": "Brazil",
+    "runners-up": "Italy",
+    "third":"Sweden",
+    "fourth":"Bulgaria"
+  },
+  {
+    "year": "1998",
+    "organizer": "France",
+    "winner": "France",
+    "runners-up": "Brazil",
+    "third":"Croatia",
+    "fourth":"Netherlands"
+  },
+  {
+    "year": "2002",
+    "organizer": "Korea/Japan",
+    "winner": "Brazil",
+    "runners-up": "Germany",
+    "third":"Turkey",
+    "fourth":"Korea Republic"
+  },
+  {
+    "year": "2006",
+    "organizer": "Germany",
+    "winner": "Italy",
+    "runners-up": "France",
+    "third":"Germany",
+    "fourth":"Portugal"
+  },
+  {
+    "year": "2010",
+    "organizer": "South Africa",
+    "winner": "Spain",
+    "runners-up": "Netherlands",
+    "third":"Germany",
+    "fourth":"Uruguay"
+  },
+  {
+    "year": "2014",
+    "organizer": "Brazil",
+    "winner": "Germany",
+    "runners-up": "Argentina",
+    "third":"Netherlands",
+    "fourth":"Brazil"
+  },
+  {
+    "year": "2018",
+    "organizer": "Russia",
+    "winner": "France",
+    "runners-up": "Croatia",
+    "third":"Belgium",
+    "fourth":"England"
+  },
```

Comment:
Extra comma here by accident 😄

Reply:
copy-paste mistake D:
I'll solve it, thanks!